TombEngine/Game/Lara/lara_fire.cpp

```diff
@@ -386,9 +501,7 @@ void LaraGun(ItemInfo* laraItem)
 	}
 
 	if (laraItem->HitPoints <= 0)
-	{
```

Comment:
Removing brackets in this case decreases readability, I think I will add a new rule to coding conventions for this particular case. Can you revert it?

Reply:
Sure, I can adapt.

Reply:
Thanks, I have also added a point to #294 about that.